Sandbox/DocumentationSnippets/FiltersApi/src/index.js

```diff
@@ -0,0 +1,15 @@
+import { initSciChart} from "./scaleoffset1";
```

Comment:
I would recommend to use prettier (as i did in this project https://github.com/ABTSoftware/SciChart.JS.Examples/tree/dev_v2.x/Sandbox/DocumentationSnippets/StylingLoader), this will make the code look nicer